source/about_jena/contributions.md

```diff
@@ -11,7 +11,7 @@ This list is provided for information purposes only, and is not meant as an
 endorsement of the mentioned projects by the Jena team.
 
 If you wish your contribution to appear on this page, please raise a
-Jira issue with the details to be published.
+Jira or GitHub issue with the details to be published.
```

Comment:
Slightly better to github first as it's the main channel now.

Reply:
Done